model_compression_toolkit/qat/keras/quantizer/ste_rounding/symmetric_ste.py

```diff
@@ -28,6 +28,11 @@
 from model_compression_toolkit.qat.common.constants import FQ_MIN, FQ_MAX
 from model_compression_toolkit import quantizers_infrastructure as qi
 from model_compression_toolkit.core.common import constants as C
+from model_compression_toolkit.quantizers_infrastructure.keras.inferable_quantizers.weights_inferable_quantizers.weights_pot_inferable_quantizer import WeightsPOTInferableQuantizer
```

Comment:
consider importing all the quantizers in the inferable_quantizers __init__, so you can import them all from the same location